### DISCUSSION TITLE
ایجاد پلتفرم برای نشر مانیفست جنبش

### DISCUSSION THREAD

**Discussion by ksaffarian** [WeAreMahsaAmini/Core#388] [Ideas]
سلام
این جنبش با آرمان های مثبت که همگی مطابق حقوق اولیه انسان هاست، ایجاد شده است.
متاسفانه، در این بزنگاه، گروه های خاصی میخواهند علایق و سلایق خود و بخصوص اهداف غیر منطقی و بعضا متضاد با آرمان اولیه را در این جنبش پیگیری کرده و این موضوع بعضا ممکن است منجر به انحراف در این حرکت خودجوش گردد.
از طرفی، حکومت، با چسباندن وصله های متعدد به این جنبش از قبیل تجزیه طلبی، منافق، تروریست و ...، در راستای تطهیر خود و خروشاندن حامیان خود به مبارزه متقابل است.
به نظر من، یک مانیفست، که اهداف اصلی و خواسته های بنیادین جنبش را نمایان سازد، میتواند به تداوم این راه، و خط کشی روشن بین خودی و غیر خودی کمک کرده و مانع از هجوم مخالفان، به بهانه های واهی گردد.
البته، من آدم سیاسی نیستم و سواد سیاسی هم ندارم، و اگر فکر‌ میکنید اینکار ممکن است باعث تغییر مفهوم اسم رمز "مهسا امینی" و آرمان های "زن، زندگی آزادی"، "مرد، میهن، آبادی" شود، بهتر انجام نشود.
لذا، صرفا این موضوع را پیشنهاد میکنم تا ببینم نظر دوستان چیست و چه پیشنهادی دارند. در صورت موافقت، وجود یک پلتفرم برای تدوین، نشر و نگهداری از این مانیفست، میتواند خروجی کار باشد.
با سپاس

**Comment by bidar-ca** [comment]
سلام 
به نظر من هم ایده شما ضروری به نظر میرسه 

لطفا #209  را هم ببینید

**Comment by hosseinkhojany** [comment]
سلام 
این ایده باید ساخته بشه و باید در زیر ساخت های غیر متمرکز و ضد فیلتر ان کار شود بشه

**Comment by AnonymousMaximus2** [comment]
مانیفست پیشنهادی ما:

فصل اول خلع سلاح دشمن

1- برای مبارزه با جمهوری اسلامی ایران و داعشی های جمهوری اسلامی ایران و هر دشمن مشابه اینها باید دشمن را خلع سلاح کرد، و برای خلع سلاح این موجودات، مهم ترین سلاح شان که اسلامی است که از انسانهای صلح طلب دزدیده اند را از اینها گرفت تا قدرت اصلی شان گرفته شود.


2- از آنجائی که 2 میلیارد نفر در دنیا مسلمان هستند، و اسلام کلا دین منفی نیست و در بان عربی به معنی صلح و آرامش و دوستی و یکی بودن است، و جمهوری اسلامی با ادعای دروغین اسلام از آن سوء استفاده نموده اند، لذا باید پتانسیلهای اسلام و مردم مسلمان راستین و صلح طلب را نگه داشته و از پتانسیل 2 میلیارد نفر جمعیت اسلامی جهان استفاده کرد و اسلام دروغینی که امثال داعش و حکومت ایران از آن دم میزنند را با ابزارهای رسانه ای، روانی، فرهنگی، هنری، و اطلاع رسانی به مردم شناسانید و از قدرت 2 میلیارد نفر جمعیت اسلامی جهان بر علیه مدعیان دروغین اسلام استفاده کرده و آنها رادخلع سلاح کرد . 


فصل دوم اتحاد مردم دنیا

1- ریشه جنگ های جهانی: برای اتحاد و صلح بین مردم دنیا باید ریشه تفرقه و جنگی که عده ای مسلمان دروغین بین مردم دنیا تفرقه و دشمنی ایجاد کرده اند را شناخت. و این ریشه عمدتا در بین ادیان و تحریف حقیقت و واقعیت ادیان اتفاق افتاده است. و اکثر کشته شده گان طول تاریخ توسط تحریف کنندگان ادیان مانند اسلام انجام شده است.

2- تعریف اسلام: اسلام کلمه ای عربی و به معنی صلح و آرامش و در امان و امنیت نگه داشتن خود و جامعه است.

3- اصول و فروع اسلام: 

1-3- اصول اسلام 5 تا است، یکی بودن خدا، حتی آن شخصی که خدا را باور ندارد نیز در درونش خودش را که ریشه از آگاهی درونی دارد نیز حس میکند و درک میکند پس او نیز خدای درون خودش و آگاهی خودش را قبول دارد. پس حتی افرادی که خدایان تحریف شده ادیان را قبول ندارند نیز به یگانگی آگاهی و درون خودشان باور دارند و اینها نیز خداشناس هستند و لو خدایان تحریف شده و ساختگی ادیان را قبول نداشته باشند. پس ساختن خدایی که مدعیان اسلام و دزدهای اسلامی از آن دم میزنند یکی از ابزارهای فریب مردم برای سوء استفاده از قدرتهای جوامع است.

3-2- اصل دیگر اسلام نبوت و پیامبری است که در قران آمده است که تمامی پیامبران یکسان و برابر هستند و فرقی بین پیامبران وجود ندارد یعنی که پیامبر اسلام با عیسی مسیح و حضرت موسی همه باهم برابر هستند و کتاب شان نیز همه مثل هم است (سوره 2 آیه 285 و 136 و سوره 3 آیه 84 ) و همچنین  در سوره 18 آیه 110 و سوره 41 آیه 6 اشاره میکند که پیامبر اسلام یک بشر مانند تمامس مردم دیگر است و فرقی با مردم دیگر ندارد به جز اینکه به ایشان وحی میشود، و در بخش دیگری از قران نیز در سوره 16 آیه 68 به وحی به زنبور عسل اشاره میکند،  پس اگر به زنبور عسل وحی میکند پس به کلیه انسانها که هوش و آگاهی بالائی دارند نیز میتواند وحی کند.  پس قداست سازی و فاصله انداختن بین پیامبران از ابزارهای دزدی اسلام است.

3-3- معاد یکی دیگر از اصول اسلام است و اینکه در قران آمده است که جهنم و بهشت همین الان در این لحظه حال حضور دارد، و در درون انسانها و از اعمال آنها سرچشمه میگیرد و جهنم و بهش جائی در آن طرف یا در دنیایی دیگر یا در دنیای بعد از مرگ نیست و این موارد با صلح خداوندی و آیه های قران در تضاد بوده و تحریف کنندگان اسلام و حکومت هایی مانند جمهوری اسلامی از تعریف من درآوردی از جهنم و بهشت برای یارگیری ارتش و سلاح سیاسی و اجتماعی و نظامی خود سوء استفاده میکند.

3-4- عدل و عدالت یکی دیگر از اصول اسلام است ولی از آنجائی که نمیتوانند مانند بهشت و جهنم و خدای من در آوردی عدل را به بازی بگیرند لذا کارهایی که انجام میدهند را به عدل الهی و عدالت دینی نسبت داده و از قدرت های جامعه بعنوان سلاحی برای قدرت گرفتن خودشان سوء استفاده میکنند.

3-5- امامت : امامت که در زبان عربی به معنی رهبری میباشد در تمامی ملتها حتی در بین آنهائی که مسلمان نیستند نیز وجود دارد، حتی امروزه بنگاههای اقتصادی نیز در سازمان خود یک رهبر و امام دارند ولی متاسفانه مدعیان اسلام تحریف شده و دروغین امامهای مخصوصی را بسیار در حد بت بزرگ کرده و امامی را نیز به غیبت برده و خودشان را نماینده آن امامی که غایب است معرفی کرده و حتی حکومت جمهوری اسلامی ایران با نظریه جعلی و ساختگی ولایت فقیه که فقط خودشان قبول دارند، تا زمانی که آن امام و رهبری که غیب نموده است خودش را نشان ندهد، حکومت اسلامی ولایت فقیه را در میان مسلمانانی که دانش عمیق از اسلام ندارند جعل نموده است. 

4- اصول دین را هر کسی باید خودش در درون خودش جستجو و به نتیجه برسد و هیچ کس حق ندارد اصول دین خودش را برای دیگری اجبار کند و این بر خلاف اسلام است ولی جمهوری اسلامی ایران در اسلام تحریف شده و من درآوردی خود، حتی برای افرادی که اصول دین آنها را انکار کنند حکم اعدام نیز تعیین کرده است.

5- ادیان آسمانی مانند مسیحیت، و کلیمی (یهود) و نصرانی (پیروان داوود پیامبر) و سایر ادیان مانند زرتشتی ها دارای کتاب هستند و به خدای واحد ایمان و اعتقاد دارند و تقریبا با کمی دقت تمامی اصول دینی که اسلام تعریف کرده با کمی جزئیات همه چنین اصولی را دارند و مسلمانان حتی آن پیامبران و کتابشان و خدائی که اعتقاد دارند را نیز طبق آیان قران باور دارند. پس مسلمانان به نوعی مسیحی و یهودی نیز حساب میشوند. و اختلاف افکندن بین مسلمانان و مسیحیان و یهودیان به دلیل سوء استفاده از مردم دنیا برای تسلط به آنها است. 

6- اسلام دارای چندین فرع میباشد که از جمله نماز روزه ، حج ، جهاد، خمس و زکات ، پیشنهاد به خوبیها (امر به معروف)، یاد آوری مضرات بدیها (نهی از منکر)  ، دوست داشتن خوبیها (تولی) بیزاری از بدی ها و چیزهایی که برای انسان و زندگی انسان ضرر دارد (تبری) که متاسفانه مدعیان ادیان و اسلام از جمله حکومت جمهوری اسلامی ایران با تفسیر به رای و تعبیر های سلیقه ای از یک طرف و اشتباه انداختن مردم در فرق بین اصل دین با فرع دین، به طور کلی چهره ای کریه و بسیار منزجر کننده از اسلام و دین دوستی و صلح نشان داده اند تا به قول معروف تفرقه بیاندازند و حکومت کنند. 

7- نه در اصول دین و نه در فروع دین نامی از حجاب برده نشده است بلکه این افراد با تحریف کردن معنی و مفهوم امر به معروف که فرعی از دین میباشد، حجاب را ابزاری برای شکنجه و کشتار مردم قرار داده اند.

8-  در قران فقط در 2 مورد دستور به جنگ و حمله و کشتار داده شده است و آن در مورد کافران و دوم در مورد طاغوت است، کافر یعنی کسی که اصول زندگی اجتماعی و صلح و آرامش بشری را زیر سوال میبرد و لو نام خودش را مسلمان گذاشته باشد. کافر کسی است که قدر دان زحمات مردم نیست و مردم از دست او در امنیت و آرامش نیستند. کافر در زبان عربی یعنی کسی که همه خوبیها را انکار میکند و چنین کسی قطعا بیمار روانی است که در سطح پائین در جامعه دستگیر و روانه زندان میشود تا مردم امنیت و آرامش داشته باشند و در سطح گسترده که به صورت یک حکومت و قدرت نظامی باعث کشتار مردم میشود با نام طاغوت یاد شده است که نه فقط قران بلکه امروزه تمامی مردم دنیا میدانند که باید بر علیه چنین حکومت خطرناکی قیام کرده و آن را نابود کرد.

9- در قران در هیچ جا علیه یهودیان، مسیحیان، و یا حتی مردمی که خدای ساختگی ادیان تحریف شده را باور ندارند دستور به حمله و کشتار داده نشده است و تمامی این حرفهای تحریف شده و من درآوردی ابزاری برای سوء استفاده حکومتهایی است که اسلام را دزدیده و با نام اسلام دیکتاتوری خود را میخواهند بر دنیا مسلط کنند.

10-  در قران و اسلام اشاره ای که کشته شدن عیسی مسیح توسط یهودیان نشده است تا پیروان حضرت موسی (یهودیان و کلیمیان) با پیروان عیسی مسیح جنگ و کشتار و تفرقه نداشته باشند.

11- در قران خدای مسیحیان یکی ذکر شده تا با خدای مسلمانان و خدای یهودیان یکی باشد و اختلافی مابین این ادیان نباشد تا بتوانند در صلح و آرامش در کنار یکدیگر زندگی کنند، ولی طاغوت جمهوری اسلامی ایران، و طاغوت سایر ادیان با تحریف کردن حقیقت و داستان جعلی کشته شدن عیسی مسیح توسط یهودیان جنگی بین یهودیان و مسیحیان انداخته و از سوی دیگر با سه گانه تعریف کردن خدای مسیحیان جنگی دیگر بین مسلمانان و مسیحیان ایجاد کرده اند تا در دنیا بتوانند با تفرقه انداختن بین ادیان حکومت طاغوتی خود را گسترش بدهند.

12- تمامی ادیان و در آخر از همه اسلام، هزاران سال پیش زمانی به مردم پیشنهاد شدند که مردم علم و دانش کافی نداشتند و لذا آن زمان مجموعه ای از قوانین نظم در جامعه و بهداشت فردی، و جسمی و سلامت روانی را با نام دین به مردم پیشنهاد دادند. ولی امروز این دانشها گسترده شده و آنقدر پیشرفت کرده اند که هر کدام یک رشته های تخصصی در دانشگاه و هر رشته میلیونها کتاب و تحقیق برایش انجام شده است . به عبارت دیگر، دین اسلام یا دین مسیحیت و یا دین کلیمی یهودیان، امروز به بلوغ رسیده و قوانین و مقررات و حقوق اجتماعی و نظم جامعه به صورت قانون توسط دولت ها اجراء میشود و بهداشت فردی و روانی و جسمی توسط علوم پزشکی و سلامتی به مردم آموزش داده میشود  و همینطور تمامی دانش بشری به صورت یک بسته جامع دین مردم امروز کره زمین است. جوهره آن همان است که هزاران سال پیش پیامبران به بشر آموزش و تعلیم دادند. و لذا دانش بشری تفسیر و توضیح کامل اسلام و مسیحیت و کلیمی است که در طول تاریخ به بلوغ رسیده و تحریفات بسیاری از آنها پاک شده است. اما متاسفانه هنوز تحریفات و سوء استفاده های بسیار زیادی در داخل قوانین حکومتها و دولتها و همچنین در داخل ادیان وجود دارد که باید با صلح و آرامش و صلح و دوستی و پیشرفت علم یک یک به بلوغ رسیده و برای بشر رشد و رفاه و آرامش و صلح به ارمغان خواهد آورد و امروز نباید به بهانه دین به طاغوتها فرصت سواری گرفتن از ملتها و پیروان ادیان بدهیم.


13-  زمانی که خلع سلاح اصلی مدعیان دروغین اسلام انجام شد، شکست و نابودی و انزوای آنها برای همیشه در تاریخ بسیار آسان خواهد بود



بقیه را هم شما تکمیل کنید.

**Comment by AnonymousMaximus2** [comment]
ساختار کلی و فونداسیون مانیفسم را ما نوشتیم،

لطفا بقیه دوستان و بزرگان به جای ایراد گرفتن، این متن را ویرایش کنند و کمک کنند کاملتر و از نظر جمله بندی و لغات و عبارات و تعاریف کاملتر و غنی تر شده و سپس هر کسی دوست داشت در مانیفست خودش و گروه های همراه خودش استفاده کند.

این نظری است که تعدادی از افراد سالها روی آن کار کرده اند، و به نظر ما تضمین کننده صلح جهانی و امنیت مردم دنیا است 

اگر ایرادی دارد لطفا همکاری کنید تا اصلاح بشود. 

لطفا سعی نکنید جبهه گیری و دشمنی و انتقاد دیکتاتور گونه انجام بدهید،

 اگر نظری و جملاتی بهتر و واضح تر و شفافتر و گویاتر و دارای صلحی عمیق تر دارید بنویسید تا استفاده کنیم.

**Comment by ghost** [comment]
#no_to_sedition
#نه_به_آشوب
#نه_به_اغتشاش
#صلح
#مرگ_بر_آشوبگر

**Reply by AnonymousMaximus2** [reply]
تکبیییییر :)

اون مهدی 313 بزنه توی سرت که توی 1400 سال از زمان اسلام تا الان 313 نفر آدم پیدا نکردی تا امام زمانت ظهور کنه و امثال تو را از منجلاب طاغوت جمهوری اسلامی نجات بده

**Reply by ghost** [reply]
> #no_to_sedition #نه_به_آشوب #نه_به_اغتشاش #صلح #مرگ_بر_آشوبگر

اتفاقا باهاتون کاملا موافقم
هیچ جای دنیا درست کردن خرابی و آشوب کار درستی نیست
کسانی که این کارها رو میکنند ظاهرا از بچگی مفهوم اموال عمومی رو به درستی یاد نگرفتند
آرزو میکنم که همچین موجوداتی دیگه روی زمین نباشند

**Reply by ghost** [reply]
اون دوست مون هم که 313 رو نقد کرده خواهشمندم یکم بره و مطالعه کنه و فعلا صحبت نکنه
